docs/dev/code_review.md

```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it essential to ensure the followig: 
+
+- code is user readable
+- follows consistency
+- helps create collaboration
+- structure is proper
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN. 
+Doing code reviews can also be good learning experiences to better understand the software and how to program. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes. 
+If this is the case, then code review parts or specific files to reduce your time spent. 
```

Comment:
```suggestion
If this is the case, then you can code review parts of a file or specific files to reduce your time spent. 
```

Reply:
Could add a 'Every little helps!' on the end?
```suggestion
If this is the case, then you can code review parts of a file or specific files to reduce your time spent. Every little helps!
```
Two suggestions so its easier than you having to open the file and change :P